src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -923,7 +921,7 @@ class ADIOSWriter : public ILightweightPlugin
                         size_t index_src = base_index_src + (x + field_guard[0]) * nComponents + d;
                         size_t index_dst = base_index_dst + x;
 
-                        params->fieldBfr[index_dst] = ((float_32*)ptr)[index_src];
+                        fieldBfr[index_dst] = ((ComponentType*)ptr)[index_src];
```

Comment:
Signal: Segmentation fault (11)
Signal code: Address not mapped (1)
Failing at address: (nil)